server/routes/visit.ts

```diff
@@ -84,11 +84,12 @@ export default function routes(
         location: prisoner.locationDescription,
       },
       visit: {
-        id: visit.reference,
+        id: '',
```

Comment:
id is not reference, this id is generated by the code and is always from 1 upwards depending on available sessions.